### PR TITLE
Added support for Toolchains

### PR DESCRIPTION
**Pull request by DreierF**
Hi Paul,
This PR implements the feature request outlined in #47.

Let me know what you think :)

### CODE REVIEW

File: src/main/kotlin/com/github/psxpaul/task/JavaExecFork.kt [hunk 1]
```diff
@@ -78,4 +90,11 @@ open class JavaExecFork @Inject constructor(forkOptionsFactory: JavaForkOptionsF
         val maxCommandLineLength = if (DefaultNativePlatform.getCurrentOperatingSystem().isWindows()) 32767 else 131072
         return args.joinToString(" ").length > maxCommandLineLength
     }
+
+    private fun getEffectiveExecutable(): String {
```
Comment (DreierF) [inline src/main/kotlin/com/github/psxpaul/task/JavaExecFork.kt:94]: Analog to https://github.com/gradle/gradle/blob/master/subprojects/language-java/src/main/java/org/gradle/api/tasks/JavaExec.java#L777

File: README.md [hunk 2]
```diff
@@ -90,4 +90,5 @@ Gradle Version | ExecFork version
 < 4.10 | 0.1.8
 4.10 - 5.2.x | 0.1.9
 5.3.0 - 5.5.x | 0.1.11+
-5.6.0+ | 0.1.12+
\ No newline at end of file
+5.6.0 - 6.6.x | 0.1.12 - 0.1.15
+6.7.0+ | 0.1.16+
```
Comment (DreierF) [inline README.md]: `JavaLauncher` was introduced with Gradle 6.7

**Comment by psxpaul** [comment]
Hey thanks for the PR, and sorry for the delay - I rarely check github these days. I think the version should be 0.2.0 if this introduces incompatibility with gradle 6.x. Also, can you please add a test project in the `sample_projects` directory?

**Comment by DreierF** [comment]
Thanks for the review!

Reworked accordingly

**Review by psxpaul — APPROVED** [review]

**Pull request merged**